plugins/webp-uploads/picture-element.php

```diff
@@ -44,12 +44,13 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 	/**
 	 * Filter the image mime types that can be used for the <picture> element.
 	 *
-	 * Default is: ['image/avif', 'image/webp', 'image/jpeg']. Returning an empty array will skip using the `picture` element.
+	 * Default is: ['image/avif', 'image/webp']. Returning an empty array will skip using the `picture` element.
 	 *
 	 * The mime types will output in the picture element in the order they are provided.
 	 * The original image will be used as the fallback image for browsers that don't support the picture element.
 	 *
 	 * @since 2.0.0
+	 *
```

Comment:
maybe annotate the default change?
```suggestion
	 * @since n.e.x.t. The default value was updated, removing 'image/jpeg'.
	 *
```

Reply:
Added in https://github.com/WordPress/performance/pull/1408/commits/9ddf45a72d88ae466c66ac23e2e84c7d60230b91. FYI we don't need last period in `@since n.e.x.t`